### PR TITLE
Added guidelines for support

### PR DESCRIPTION
**Pull request by codenamewei**

### CODE REVIEW

File: SupportGuidelines.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+**3. Environment Setup**
+- Intention / Motivation of the action.  
+Identify what's the intention of the action or which program's user wants to run.  
+This can narrow down the scope of searching.  
+  
+On some occasions, we can also suggest a better way of execution after truly understands the intention behind the action of doing something.
+
+### Optional Step 2: Provide Workaround
+If could not identify problem through the first step, focus on suggest the workarounds to solve the immediate pain,  
+   then spend time in replicating the problem on own system (in order to start solving the problem).
```
Comment (YCCertifai) [inline SupportGuidelines.md:73]: I see it's repeated up there..
is that insisted?

File: SupportGuidelines.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+1. **Get as much of factual information surrounding the problem.** 
+    - Operating System
+    - Software Version
+    - Specific Program to Run
+    - Environment Setup
+    - Intention / Motivation of the action
+    
+2. **Provide workaround.**  
+If could not identify problem through the first step, focus on suggest the workarounds to solve the immediate pain,  
+   then spend time in replicating the problem on own system (in order to start solving the problem).
```
Comment (YCCertifai) [inline SupportGuidelines.md:13]: here.

Comment (codenamewei) [reply SupportGuidelines.md:13]: @YCCertifai yeah, its intention for people to just read the upper parts if they do not want to read the whole article.

often it quoted with [TLDR](https://www.google.com/search?q=TLDR%3B&rlz=1C5CHFA_enMY930MY930&oq=TLDR%3B&aqs=chrome..69i57j69i59j69i58.2359j1j4&sourceid=chrome&ie=UTF-8)

Comment (YCCertifai) [reply SupportGuidelines.md:13]: ok will merge it

**Pull request merged**